db/migrate/20211224082345_add_threshold_for_debate_to_petitions.rb

```diff
@@ -0,0 +1,24 @@
+class AddThresholdForDebateToPetitions < ActiveRecord::Migration[6.1]
+  def up
+    add_column(:petitions, :threshold_for_debate, :integer)
+
+    result = execute(
+      "SELECT threshold_for_debate
+      FROM sites
+      LIMIT 1;"
+    )
+
+    threshold_for_debate = result.getvalue(0, 0)
+
+    exec_query(
+      "UPDATE petitions
+      SET threshold_for_debate = $1;",
+      "SQL",
+      [[nil, threshold_for_debate]]
+    )
+  end
+
+  def down
+    remove_column(:petitions, :threshold_for_debate)
+  end
```

Comment:
Yes, the signature count graph now gets the threshold from the petition rather than the site, so updating the petition value seems to fix this scenario 🎉 

I'll make a note to make sure we're sure they're happy with it!